rules/nod-structures.yaml

```diff
@@ -698,59 +698,6 @@ NAPYRA:
 		CircleSequence: circles
 		ClockSequence: clock
 
-NATECH:
```

Comment:
any specific reason for removing those? normally I wouldn't mind as they're unused, but if somebody placed them in a map and they're gone now then it's gg for mapmaker if he doesn't know that and how to remove actors via yaml